sormas-backend/src/main/java/de/symeda/sormas/backend/contact/Contact.java

```diff
@@ -573,6 +575,15 @@ public void setExternalToken(String externalToken) {
 		this.externalToken = externalToken;
 	}
 
+	@Column(length = COLUMN_LENGTH_DEFAULT)
```

Comment:
Columns of type text should have `@Column(columnDefinition = "text")` not `@Column(length = COLUMN_LENGTH_DEFAULT)`